### PR TITLE
Feature/login UI

### PR DESCRIPTION
**Pull request by jukatz**
I converted the alert login process into a storyboard
Refactored some of the alert messaging code

### CODE REVIEW

File: PeapodSwipe/AuthViewController.swift [hunk 1]
```diff
@@ -55,152 +53,89 @@ class AuthViewController: UIViewController {
...
+    private func isValidEmail(emalAddress: String) -> Bool {
+        let emailRegEx = "[A-Z0-9a-z._%+-]+@[A-Za-z0-9.-]+\\.[A-Za-z]{2,64}"
 
-    @objc func SELSignInWithEmail() {
+        let emailTest = NSPredicate(format: "SELF MATCHES %@", emailRegEx)
+        return emailTest.evaluate(with: emalAddress)
+    }
+    private func isDebugMode() -> Bool {
+        let args = ProcessInfo.processInfo.arguments
+        return (UserDefaults.standard.bool(forKey: "FASTLANE_SNAPSHOT") || args.contains("UI_TEST_MODE"))
```
Comment (soleo) [inline PeapodSwipe/AuthViewController.swift]: The purpose for those flags are not for debugging, they are for running UI Tests, and generating Marketing Screenshots

Comment (jukatz) [reply PeapodSwipe/AuthViewController.swift]: Updated to ```private func isTestMode() -> Bool { ... }```

File: PeapodSwipe/AuthViewController.swift [hunk 2]
```diff
@@ -55,152 +53,89 @@ class AuthViewController: UIViewController {
...
+    }
+    private func clearTextFields() {
+        emailField.text = ""
+        inviteCodeField.text=""
```
[the remark this answers is not in the document]
Comment (jukatz) [reply PeapodSwipe/AuthViewController.swift]: Thank you for catching this!

**Comment by jukatz** [comment]
@soleo The re-design is ready for your review again. Thanks!

**Comment by jukatz** [comment]
@soleo Third time is the charm!

**Pull request merged**